projects.json

```
@@ -4620,5 +4620,39 @@
         "action": "TestSwiftPackage"
       }
     ]
+  },
+  {
+    "repository": "Git",
+    "url": "https://github.com/apple/swift-testing",
+    "path": "swift-testing",
+    "branch": "main",
+    "maintainer": "bcroom@apple.com",
+    "compatibility": [
+      {
+        "version": "5.0",
+        "commit": "f9deef4f09f85064a542335b00f35430e8bf7432"
+      }
+    ],
+    "platforms": [
+      "Darwin",
+      "Linux"
+    ],
+    "actions": [
+      {
+        "action": "BuildSwiftPackage",
+        "configuration": "release",
+        "build_tests": "true",
+        "build_tests_release": "true",
+        "tags": "swiftpm",
+        "xfail": [
+          {
+            "issue": "https://github.com/apple/swift/issues/69096",
+            "compatibility": ["5.0"],
+            "branch": ["release/5.10", "main"],
```

Note that our current expected failure is with `main` toolchains only, not 5.10 toolchains. Should this be adjusted to the following? (Apologies if I'm misinterpreting what this means)
```suggestion
            "branch": ["main"],
```

---

Will update 👍